test/repl/proxysql_repl_tests/bin/debezium-repl-config.bash

```
@@ -0,0 +1,32 @@
+fi
+
+echo -n "Configuring connection: mysql1(source) => proxysql => debezium(replication stream) ..."
+
+echo -n "Waiting for kafka_connect to be ready ..."
+for _i in $(seq 1 60); do
+    curl -s -f http://kafka_connect.${INFRA}:8083/connectors >/dev/null 2>&1 && break
+    echo -n "."; sleep 2
+done
+echo " ready."
+
+conn_mysql=`envsubst < conf/debezium/register-proxysql.json`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)